grakn-graph/src/test/java/ai/grakn/graph/internal/PostProcessingTest.java

```diff
@@ -148,7 +150,7 @@ public void whenMergingDuplicateResourcesWithRelations_EnsureSingleResourceRemai
 
         assertNotNull(foundR1);
         assertThat(foundR1.ownerInstances(), containsInAnyOrder(e1, e2, e3));
-        assertEquals(4, graknGraph.admin().getMetaRelationType().instances().size());
+        assertEquals(3, graknGraph.admin().getMetaRelationType().instances().size());
```

Comment:
This is correct due to the "fakeness" of the test. The resource type is not linked properly for example